src/OmniSharp.DotNetTest/TestFrameworks/MSTestFramework.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace OmniSharp.DotNetTest.TestFrameworks
+{
+    internal class MSTestFramework : TestFramework
+    {
+        public override string FeatureName { get; } = "MSTestMethod";
+        public override string Name { get; } = "mstest";
+        public override string MethodArgument { get; } = "--test";
+
+        protected override bool IsTestAttributeName(string typeName)
+        {
+            return typeName == "Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute"
+                || typeName == "Microsoft.VisualStudio.TestTools.UnitTesting.DataTestMethodAttribute";
```

Comment:
`DataTestMethodAttribute` is not needed here, because it inherits from `TestMethodAttribute`.
The code in OmniSharp checks the whole inheritance tree for the attributes.